pokedex/SearchOptions.js

```diff
@@ -0,0 +1,45 @@
+import Component from '../Component.js';
+
+class SearchOptions extends Component {
+
+    onRender(form) {
+        const searchInput = form.querySelector('input[name=search]');
```

Comment:
lines 6 - 18 aren't actually doing anything. I could take all this code out and the app would work the same. This code was needed in the demo but not for your code.